app/views/foreman_tasks/tasks/dashboard/_latest_tasks_in_error_warning.html.erb

```diff
@@ -11,7 +11,7 @@
       <td class="ellipsis"><%= link_to task.humanized[:action], defined?(main_app) ? main_app.foreman_tasks_task_path(task.id) : foreman_tasks_task_path(task.id) %></td>
       <td><%= task.state %></td>
       <td><%= task.result %></td>
-      <td><%= _('%s ago') % time_ago_in_words(task.ended_at) %></td>
+      <td><%= task.ended_at ? _('%s ago') % time_ago_in_words(task.ended_at) : _('Paused') %></td>
```

Comment:
I was trying to figure that out as well but didn't see anything obvious. The best would be a way to grab the last updated time on the task but I don't know if thats possible. We would also need to change the "Ended" col to "Updated" on the table if we find out how to get this info.

I would vote 👍 to "N/A" if finding the updated timestamp is not possible.